apis/workloads/v1alpha1/lifecycle_recorder.go

```diff
@@ -52,18 +52,24 @@ func NewLifecycleEventRecorder(contextName string, logger logr.Logger, publisher
 	}
 }
 
-func (l *lifecycleEventRecorderImpl) ConsoleRequest(ctx context.Context, csl *Console) error {
+func (l *lifecycleEventRecorderImpl) ConsoleRequest(ctx context.Context, csl *Console, authRule *ConsoleAuthorisationRule) error {
+	var authCount int
```

Comment:
```suggestion
	authCount := 0
```

I think it might be better to be explicit about the default value we're expecting here.